src/test/groovy/func/GatlingRunTaskTest.groovy

```diff
@@ -32,9 +33,9 @@ class GatlingRunTaskTest extends GatlingUnitSpec {
 
     def "should resolve simulations using custom filter"() {
         given:
-        project.gatling.simulations = { include "**/*AdvancedSimulation*" }
+        project.gatling.includes = [ "**.*AdvancedSimulation*" ]
```

Comment:
Not sure to understand.

Before, it was glob syntax. now it is regex syntax?
If yes:
```suggestion
        project.gatling.includes = [ ".*AdvancedSimulation.*" ]
```
But I guess, it's only the end we want:
```suggestion
        project.gatling.includes = [ ".*AdvancedSimulation" ]
```
As before we had the ".java" extension to accepts, but now, we have the class name.

Reply:
Note, it means this is a breaking change!

Reply:
Now, it's ant patterns (I think plexus-util also supports regex, but you need to add a prefix. We won't advertise it).

> Note, it means this is a breaking change!

Absolutely. Before the filters were working on source files, now they are working on class names.